.github/workflows/go.yml

```
@@ -75,6 +75,7 @@ jobs:
   benchmark:
     name: Benchmark
     runs-on: ubuntu-latest
+    if: false
```

actionlint (1.7.9)

condition "false" is always evaluated to false. remove the if: section

(if-cond)